src/main/resources/de/itemis/mps/gradle/bundle_macos_jdk.sh

```diff
@@ -1,16 +1,17 @@
 #!/bin/bash
-if [[ $# -ne 4 ]]; then
+if [[ $# -ne 5 ]]; then
   cat <<EOF
 Usage:
 
-  $0 RCP_FILE TMP_DIR JDK_FILE OUT_FILE
+  $0 RCP_FILE TMP_DIR JDK_DIRNAME JDK_FILE OUT_FILE
 
 1. Extracts RCP_FILE into TMP_DIR
 2. Creates symlinks Contents/bin/*.dylib -> *.jnilib if any .jnilib files are present
    (this used to be the case for earlier versions of MPS)
-3. If JDK_FILE is given, extracts JDK_FILE under Contents/jre/
-4. Sets executable permissions on Contents/MacOS/* and appropriate Contents/bin/ files
-5. Compresses the result into OUT_FILE (tar/gzip)
+3. JDK_DIRNAME to use if JDK_FILE is given
```

Comment:
This is not a description of arguments, but a list of steps that the script performs.

Reply:
Cool, just removed my added step since the subsequent amended step was sufficient